cartography/data/jobs/cleanup/azure_tenant_cleanup.json

```diff
@@ -1,9 +1,10 @@
 {
 	"statements": [
-	{
-		"query": "MATCH (n:AzureTenant{id: {AZURE_TENANT_ID}}) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
-		"iterative": true,
-		"iterationsize": 100
-	}],
+		{
+			"query": "MATCH (n:AzureTenant) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
we should not remove this from the cleanup file. instead, we should pass this as part of the cleanup configuration object.